### DISCUSSION TITLE
Simple example for using this with angular

### DISCUSSION THREAD

**Discussion by hemmoleg** [Eliav2/react-xarrows#62] [Q&A]
I would love to use this, but in an angular environment. Could someone please provide a simple codepen on how to do this?

**Comment by Eliav2** [comment]
Using this in angular won't be simple because you will need to rewrite all the 'Reactish' logic

**Comment by hemmoleg** [comment]
As it turns out it was quite easy. I just used this [tutorial](https://blog.harveydelaney.com/integrating-react-components-into-an-angular-2-project/) and it worked flawless.

**Reply by Eliav2** [reply]
nice! good to know!
